licenses.yaml

```diff
@@ -1962,7 +1962,7 @@ name: Jetty
 license_category: binary
 module: java-core
 license_name: Apache License version 2.0
-version: 9.4.34.v20201102
+version: 9.4.38.v20210224
```

Comment:
CI failure looks like it wants `jetty-util-ajax` added to the `libraries` section

Reply:
Fixed. Thanks!